pom.xml

```diff
@@ -465,6 +465,30 @@
                             <port>8081</port>
                         </ports>
                     </container>
+                    <from>
+                        <platforms>
+                            <platform>
+                                <architecture>amd64</architecture>
+                                <os>linux</os>
+                            </platform>
+                            <platform>
+                                <architecture>arm64</architecture>
+                                <os>linux</os>
+                            </platform>
+                            <platform>
+                                <architecture>arm</architecture>
+                                <os>linux</os>
+                            </platform>
+                            <platform>
+                                <architecture>ppc64le</architecture>
+                                <os>linux</os>
+                            </platform>
+                            <platform>
+                                <architecture>s390x</architecture>
+                                <os>linux</os>
+                            </platform>
+                        </platforms>
```

Comment:
```suggestion
```

I'm happy to build arm64 but I think the rest we don't want to build for every commit.